http2/go-grpc/README.md

```diff
@@ -1,15 +1,41 @@
 # Go gRPC Sample
 
 Example gRPC server implemented in Go
+## Deploying the Go app
+### Deploy the app using the manifest file
+Set the desired domain in [vars.yml](../vars.yml) file:
+```shell
+# edit the vars.yml file and replace <domain.for.app.routes> with required domain
+# my.example.domain is used as an example for demonstration
+> vi vars.yml
+> cat vars.yml
+domain: my.example.domain
+```
+Deploy the app
+```shell
+> cf push -f app-manifest.yml --vars-file ../vars.yml
```

Comment:
```suggestion
cf push -f app-manifest.yml --var domain=$DOMAIN
```

This applies to all examples. I would really like to see the `>` gone as they make copy&paste of the examples impossible.